config/config.yaml

```diff
@@ -19,11 +20,11 @@ kraken:
         -   time_scenarios:                                # List of chaos time scenarios to load
             - scenarios/time_scenarios_example.yml
         -   litmus_scenarios:                              # List of litmus scenarios to load
-            - - https://hub.litmuschaos.io/api/chaos/1.13.6?file=charts/generic/node-cpu-hog/rbac.yaml
+            - - https://raw.githubusercontent.com/cloud-bulldozer/kraken/master/scenarios/templates/litmus-rbac.yaml
```

Comment:
Would be nice to point to the local copy of that file: scenarios/templates/litmus-rbac.yaml

Reply:
Agreed, should we skip this change in favor of #149? Thoughts?

Reply:
Sure! That works!